src/cclib/parser/orcaparser.py

```diff
@@ -62,6 +62,114 @@ def extract(self, inputfile, line):
         if "Program Version" in line:
             self.metadata["package_version"] = line.split()[2]
 
+        # ================================================================================
+        #                                         WARNINGS
+        #                        Please study these warnings very carefully!
+        # ================================================================================
+        #
+        # Warning: TCutStore was < 0. Adjusted to Thresh (uncritical)
+        #
+        # WARNING: your system is open-shell and RHF/RKS was chosen
+        #   ===> : WILL SWITCH to UHF/UKS
+        #
+        #
+        # INFO   : the flag for use of LIBINT has been found!
+        #
+        # ================================================================================
+        if line.strip() == "WARNINGS":
```

Comment:
This is interesting, I like this a lot. At some point we should think about what gets sent to the logger and what gets added to metadata (or both).

Reply:
Agreed. I am fine with a `metadata['warnings']` thing for now.